messaging/src/test/java/org/axonframework/messaging/GenericMessageTest.java

```diff
@@ -68,15 +71,22 @@ void correlationDataAddedToNewMessage() {
     }
 
     @Test
-    void messageSerialization() {
+    void messageSerialization() throws IOException{
         GenericMessage<String> message = new GenericMessage<>("payload", Collections.singletonMap("key", "value"));
         Serializer jacksonSerializer = JacksonSerializer.builder().build();
 
         SerializedObject<String> serializedPayload = message.serializePayload(jacksonSerializer, String.class);
         SerializedObject<String> serializedMetaData = message.serializeMetaData(jacksonSerializer, String.class);
 
         assertEquals("\"payload\"", serializedPayload.getData());
-        assertEquals("{\"key\":\"value\",\"foo\":\"bar\"}", serializedMetaData.getData());
+
+        ObjectMapper objectMapper = new ObjectMapper();
```

Comment:
You could retrieve the `ObjectMapper` from the `JacksonSerializer` that's constructed on line 76 by invoking `JacksonSerializer#getObjectMapper`. That would require changing the type from `Serializer` to `JacksonSerializer` there, of course.

Reply:
Yes, I've made the change in the most recent commit to take this in account.